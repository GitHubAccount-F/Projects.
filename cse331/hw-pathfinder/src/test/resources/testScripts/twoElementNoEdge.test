#Tests Dijkstra's algorithm with a graph containing only two
#nodes, but with no edges between them

#Creates the graph
CreateGraph graph1

#Adds the nodes
AddNode graph1 one
AddNode graph1 two

#Finds a path
FindPath graph1 one two

FindPath graph1 two one