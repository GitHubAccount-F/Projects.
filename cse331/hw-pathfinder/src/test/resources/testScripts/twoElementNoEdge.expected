#Tests Dijkstra's algorithm with a graph containing only two
#nodes, but with no edges between them

#Creates the graph
created graph graph1

#Adds the nodes
added node one to graph1
added node two to graph1

#Finds a path
path from one to two:
no path found

path from two to one:
no path found